src/index.html

```diff
@@ -1,13 +1,39 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="ru" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="page__content">
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="card__images"
+        class="card__images"
+      >
+      <h1 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h1>
+      <p class="card__text">Код товара: 195434</p>
+      <div class="rewiew">
+        <div class="stars stars--1">
+          <div class="stars_star"></div>
+          <div class="stars_star"></div>
+          <div class="stars_star"></div>
+          <div class="stars_star"></div>
+          <div class="stars_star"></div>
+        </div>
+        <p class="rewiew__text">Отзывов: 5</p>
+      </div>
+      <div class="price">
```

Comment:
Don't make BEM structure too complicated. You don't need to create separate blocks for anything other than card itself and stars.